### PR TITLE
Feature/71300

### PR DESCRIPTION
**Pull request by anqaka**
internal dev-71300

### CODE REVIEW

File: Snowdog_Components/components/Molecules/image/_image.scss [hunk 1]
```diff
@@ -28,3 +28,13 @@ picture {
     background-color: $image__wrapper-background-color;
     transition: $transition-base;
 }
+
+picture > img.lazyload,
+picture > img.lazyloading {
+    min-height: 56px;
```
Comment (pawel-snowdog) [inline Snowdog_Components/components/Molecules/image/_image.scss:34]: use variable

File: Snowdog_Components/components/Molecules/image/_image.scss [hunk 2]
```diff
@@ -28,3 +28,13 @@ picture {
...
+    opacity: 1;
+    background-image: url('../images/loader-svg.svg');
```
Comment (pawel-snowdog) [inline Snowdog_Components/components/Molecules/image/_image.scss:36]: use variable

File: Snowdog_Components/components/Molecules/image/_image.scss [hunk 3]
```diff
@@ -28,3 +28,13 @@ picture {
...
+    background-repeat: no-repeat;
+    background-size: 56px;
```
Comment (pawel-snowdog) [inline Snowdog_Components/components/Molecules/image/_image.scss:38]: use variable

File: Snowdog_Components/components/Organisms/slider/_slider.scss [hunk 4]
```diff
@@ -195,6 +195,13 @@
                 display: flex;
             }
         }
+
+        picture > img.lazyload,
+        picture > img.lazyloading {
+            min-height: initial;
```
Comment (pawel-snowdog) [inline Snowdog_Components/components/Organisms/slider/_slider.scss:201]: use variable

File: Snowdog_Components/components/Molecules/image/_image.scss (hunk 1 above)
Comment (anqaka) [reply Snowdog_Components/components/Molecules/image/_image.scss:34]: 👍 right, thnx, I forgot about it :)

File: Snowdog_Components/components/Molecules/image/_image.scss (hunk 2 above)
Comment (anqaka) [reply Snowdog_Components/components/Molecules/image/_image.scss:36]: 👍

File: Snowdog_Components/components/Molecules/image/_image.scss (hunk 3 above)
Comment (anqaka) [reply Snowdog_Components/components/Molecules/image/_image.scss:38]: 👍

**Review by pawel-snowdog — APPROVED** [review]

**Pull request merged**